operatorcourier/push.py

```diff
@@ -21,14 +21,18 @@ class PushCmd():
     def __init__(self):
         pass
 
-    def push(self, bundle_dir, namespace, repository, release, auth_token):
+    def push(self, bundle_dir, namespace, repository, release, auth_token,
+             quay_host, verify_host):
         """Push takes a bundle and pushes it to the specified app registry repository.
 
         :param bundle_dir: Path to generated local directory that contains the bundle.
         :param namespace: Namespace that contains the repository for the application.
         :param repository: Repository name of the application described by the bundle.
         :param release: Release version of the bundle.
         :param auth_token: Authentication token used to push to Quay.io.
+        :param quay_host: Can be 'quay.io' or your private instance hostname.
+        :param verify_host: ``quay_host`` TLS/CA verification.
+                            Either a boolean or a string.
```

Comment:
I'd put here a full documentation, or refer to requests documentation what usage of `string` means here.
From requests 
```
    :param verify: (optional) Either a boolean, in which case it controls whether we verify
            the server's TLS certificate, or a string, in which case it must be a path
            to a CA bundle to use. Defaults to ``True``.
```

Reply:
@MartinBasti Thanks! I have updated the documentation.